mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -1458,4 +1475,167 @@ class GeneralizedLinearRegressionTrainingSummary private[regression] (
         "No p-value available for this GeneralizedLinearRegressionModel")
     }
   }
+
+  /**
+   * Coefficient matrix with feature name, coefficient, standard error,
+   * tValue and pValue.
+   */
+  @Since("2.3.0")
+  lazy val coefficientMatrix: Array[(String, Double, Double, Double, Double)] = {
```

Comment:
This is not a matrix, so it's not appropriate to name it as ```coefficientMatrix```. Since it's only used for generating summary string output, what about keep it private or inline?